### DISCUSSION TITLE
WT32 ETH01 Pin out?

### DISCUSSION THREAD

**Discussion by Kampfwurst** [arendst/Tasmota#12098] [Q&A] [answered]
Hi.

Can someone tell me the settings for the WS2812 on the WT32 ETH01. I would like to use the IO2 or the IO4 pin. 
What the pin settings in the Wled interface? I tried 17 and 18 without success

**Comment by sfromis** [comment, marked as answer]
You asking for "Wled interface" sounds like the WLED project, which is entirely different from Tasmota. If you want to use the barebones neopixel support in Tasmota, that would simply be to use the Configure Module/Template to assign the desired free pin as WS2812.
